### PR TITLE
FB Pixel: version migration to v13.0

### PR DESCRIPTION
**Pull request by akashrpo**
## Description of the change

> version migration to v13.0
https://developers.facebook.com/docs/graph-api/changelog/versions

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: v0/destinations/fb/transform.js [hunk 1]
```diff
@@ -220,16 +220,11 @@ function processEventTypeGeneric(message, baseEvent, fbEventName) {
...
-    requestFormat: "FORM",
-    requestMethod: "POST"
-  };
-
   const { appID } = destination.Config;
 
   // "https://graph.facebook.com/v3.3/644758479345539/activities?access_token=644758479345539|748924e2713a7f04e0e72c37e336c2bd"
 
-  const endpoint = `https://graph.facebook.com/v3.3/${appID}/activities`;
+  const endpoint = `https://graph.facebook.com/v13.0/${appID}/activities`;
```
Comment (ItsSudip) [inline v0/destinations/fb/transform.js:227]: Have you validated sending the event to the destination as it is a long jump in respect of versions?

Comment (utsabc) [reply v0/destinations/fb/transform.js:227]: +1

Comment (akashrpo) [reply v0/destinations/fb/transform.js:227]: Yes

**Review by utsabc — APPROVED** [review]

**Comment by utsabc** [comment]
LGTM

**Review by ItsSudip — APPROVED** [review]

**Comment by sanpj2292** [comment]
A Suggestion:
Write a custom function which takes the FB version from a variable inside `fb_pixel/config.js` for both tests and development
So with the above changes, the amount of changes in test-cases will be reduced to bare-minimum in case of any version changes

You don't have to do it now but working on it in the next version change, would reduce development in the subsequent version changes

**Review by sanpj2292 — APPROVED** [review]

**Pull request merged**